### ISSUE TITLE
Security Command Center Legacy

### ISSUE THREAD

**Issue by argst274** [terraform-google-modules/terraform-example-foundation#633]
### TL;DR

Running 1.org Cloud Build Production job throws Error 400: Security Command Center Legacy has been permanently disabled as of June 7, 2021. 

### Expected behavior

Job completes without failure. 

### Observed behavior

Error: Error creating NotificationConfig: googleapi: Error 400: Security Command Center Legacy has been permanently disabled as of June 7, 2021. Migrate to Security Command Center's Standard tier or Premium tier to maintain access to Security Command Center. See https://cloud.google.com/security-command-center/docs/quickstart-security-command-center for more info.

  on scc_notification.tf line 32, in resource "google_scc_notification_config" "scc_notification_config":
  32: resource "google_scc_notification_config" "scc_notification_config" {

Error: googleapi: Error 409: Sink sk-c-logging-bq already exists, alreadyExists

  on .terraform/modules/log_export_to_biqquery/main.tf line 73, in resource "google_logging_organization_sink" "sink":
  73: resource "google_logging_organization_sink" "sink" {

Error: googleapi: Error 409: Sink sk-c-logging-pub already exists, alreadyExists

  on .terraform/modules/log_export_to_pubsub/main.tf line 73, in resource "google_logging_organization_sink" "sink":
  73: resource "google_logging_organization_sink" "sink" {

Error: googleapi: Error 409: Sink sk-c-logging-bkt already exists, alreadyExists

  on .terraform/modules/log_export_to_storage/main.tf line 73, in resource "google_logging_organization_sink" "sink":
  73: resource "google_logging_organization_sink" "sink" {

### Terraform Configuration

```hcl
Latest code base
step 4. tf apply
```

### Terraform Version

```sh
Terraform v1.1.5
on linux_amd64
```

### Additional information

n/a

**Comment by haggman** [comment]
Per the documentation on the google_scc_notification_config resource page [here](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/scc_notification_config), you need to enroll in SCC basic or premium before you can use that resource. 

I'm running through the 1-org today and I got the SCC-related error. I then went to the **Navigation menu | Security | Security Command Center** and ran through the steps to enroll in the free basic plan. After that I:
1. In the 1-org files on my laptop I updated the backend.tf to point to my bucket name.
2. I did a terraform destroy to remove all the aborted changes.
3. I went into the seed project to my Cloud Storage bucket and deleted the tfstate file in the org sub-folder, since it's now out of version sync after nuking everything from Cloud Shell, which has a very new TF version
4. Went to the Cloud Build triggers page and reran the gcp-org plan trigger. 
5. Once it finished, used the triggers page to run the gcp-org apply. 

That did the trick. Through 1-org and on to the next step!

**Comment by argst274** [comment]
@haggman - thanks for taking a look, your steps worked a treat right up until step 5! while no longer getting scc error, im still seeing the 409 alreadyExisits errors. Any ideas what im doing wrong? I dont see how they can exist after the terraform destroy removes the entire project!

**Comment by haggman** [comment]
I got the same thing. I suspect it has something to do with the projects not deleting immediately, but I'm not 100% sure. I tried again, doing steps 1-3 to nuke everything. Then I took a break for a meal, and came back and did the last two. The second time it seemed to work. 

No. Idea. Why.

Ha, but it did.

**Comment by argst274** [comment]
I gave it overnight between purge and ran the cloudbuild processes again! Apparently it hates me as im still getting the errors. 

Really appreciate your pointers tho.

**Comment by argst274** [comment]
FYI - only way to continue was to manually rename the sink names in `ens/shared/log_sinks.tf`
